backend/my_app/api/v1/views/parcel_views.py

```diff
@@ -0,0 +1,153 @@
+"""
+Creates views for parcels. These are POST, GET, PUT
+"""
+
+from flask import request, make_response, jsonify
+from flask_restful import Resource
+from ..models.parcel import ParcelModel
+
+
+class Parcels(Resource):
+    """This class provides access operation for GET and POST methods
+    for a parcel order or parcels
+    """
+
+    def __init__(self):
+        pass
+
+    def post(self):
+        """Saves a new parcel item
+        :return: Returns a json response
+        """
+        data = request.get_json() or {}
+
+        if not data or data == {}:
+            return make_response(jsonify(
+                {
+                    "Message": "Post data not provided",
+                    "status": "Bad Request"
+                }
+            ), 400)
+
+        parcel = ParcelModel()
+        parcel.add_parcel(
+            sender_id=data['sender_id'],
+            pickup_location=data['pickup_location'],
+            destination=data['destination'],
+            weight=data['weight'],
+            status=data['status'])
+
+        payload = {'status': 'Created'}
+        res = make_response(jsonify(payload), 201)
+        res.content_type = 'application/json;charset=utf-8'
```

Comment:
This is not necessary. As you have already made the response. Also, return the parcel created in your response. Same as the review for user above.